source/faq.txt

```diff
@@ -148,7 +148,7 @@ The error message consists of multiple parts:
    server description contains an exhaustive description of its current
    state including information about an endpoint, a server version, a
    server type, and its current health state. If the server is not
-   heathy, ``HeartbeatException`` contains the exception from the
+   healthy, ``HeartbeatException`` contains the exception from the
```

Comment:
Suggestion:

I think this could avoid the Vale issue if it was reworded as follows:
"If the server encounters issues in reporting its health, HeartbeatException"